src/nvim/charset.c

```diff
@@ -82,7 +82,7 @@ int buf_init_chartab(buf_T *buf, int global)
 {
   int c;
   int c2;
-  char_u *p;
+  unsigned char *p;
```

Comment:
It will effectively be removed someday AFAIK. The point that the type is not always the best and will be dealt with on time.